Extension/src/LanguageServer/client.ts

```diff
@@ -1184,6 +1186,7 @@ export class DefaultClient implements Client {
                 intelliSenseCachePath : settings_intelliSenseCachePath,
                 intelliSenseCacheSize : settings_intelliSenseCacheSize,
                 intelliSenseMemoryLimit : settings_intelliSenseMemoryLimit,
+                intelliSenseUpdateDelay: settings_intelliSenseUpdateDelay,
```

Comment:
This would be workspaceSettings.intelliSenseUpdateDelay if the scope is changed to "application".